streaming/src/main/scala/org/apache/gearpump/streaming/StreamApplication.scala

```diff
@@ -77,11 +77,25 @@ class StreamApplication(override val name : String,  inputUserConfig: UserConfig
 object StreamApplication {
 
   def apply (name : String, dag: Graph[_ <: Processor[Task], _ <: Partitioner], userConfig: UserConfig): StreamApplication = {
-    val graph = dag.mapVertex(Processor.ProcessorToProcessorDescription(_))
+    import Processor._
+    val graph = dag.mapVertex(ProcessorToProcessorDescription)
     new StreamApplication(name, userConfig, graph)
   }
 
+  def apply (name : String, jarName: String, processors: Map[ProcessorId, ProcessorDescription], dag: Graph[Int, String]): StreamApplication = {
```

Comment:
I don't think this is possible since we do not have these Processor classes available in the services JVM. We would need to load the appJar via a class loader and then instantiate each class using the same class loader.